src/xmpp/xmpp-core/connector.cpp

```diff
@@ -363,7 +363,7 @@ void AdvancedConnector::bs_connected()
     if (d->proxy.type() == Proxy::None) {
         QHostAddress h = (static_cast<BSocket *>(d->bs))->peerAddress();
         int          p = (static_cast<BSocket *>(d->bs))->peerPort();
-        setPeerAddress(h, p);
+        setPeerAddress(h, quintptr(p));
```

Comment:
quint16. well we have to review all these cases. to avoid using ints if there no other uses but just passing it somewhere.
e.g peerPort could return quint16